src/main/java/org/folio/service/transactions/batch/BatchTransactionChecks.java

```diff
@@ -65,11 +67,12 @@ public static void sanityChecks(Batch batch) {
           checkTransfer(transaction);
         }
       });
+    checkPaymentsAndCreditsAmounts(batch);
   }
 
   public static void checkBudgetsAreActive(BatchTransactionHolder holder) {
     holder.getBudgets().forEach(budget -> {
-      if (budget.getBudgetStatus() != ACTIVE) {
+      if (!List.of(ACTIVE, PLANNED).contains(budget.getBudgetStatus())) {
         throw new HttpException(400, String.format("Cannot process transactions because of an inactive budget for fund %s",
```

Comment:
error message should be updated accordingly

Reply:
OK